packages/core/src/server/index.ts

```diff
@@ -37,7 +49,17 @@ const apiOptions: APIOptions = {
   },
 }
 
-export const apiSchema = getSchema(apiOptions)
+export const nativeApiSchema = getSchema(apiOptions)
+
+const extensionsSchema = getExtensionsSchema()
+
+const getMergedSchemas = async () =>
```

Comment:
I'd like to see better error handling throughout the code. I don't know what can possibly go bad in the calls you're making to the `@graphql-tools` libraries but I believe that something might throw an exception and we better handle it here.

I know this is a draft PR, but I'm just raising the concern here :rocket:

Reply:
@graphql-tools throws errors in some schema validations already, this makes easy to get errors in dev time. 

In console: 
<img width="798" alt="Screenshot 2023-07-25 at 20 13 59" src="https://github.com/vtex/faststore/assets/11325562/86c38a98-8460-487d-a0b0-a93ab5077e95">

in the page that is not rendered:
<img width="947" alt="Screenshot 2023-07-25 at 20 14 36" src="https://github.com/vtex/faststore/assets/11325562/796a2492-4f06-4969-af94-07139a87c10f">

If we decided on handle these erros with try/catch some errors could pass and the page be rendered, unless we throw the error again in the catch. So, Handling the errors here might overlook an unwanted state, in that case. I prefer to keep showing the error in page, WDYT?

Reply:
I like the errors that graphql-tools already provide us. I'll resolve this conversation.